Kentico/devnet.kentico.com
# Deleted Sage Pay - broken link, unsupported

### Motivation

We do not want nonexistent integrations polluting our marketplace. It was for KX11 anyway which is not supported anymore.

### How to test

Visit https://www.ecommerce-connect.co.uk/Sage-Pay-Payment-Gateway-for-Kentico.aspx and see the error :)
